docs/terminal/williamtown.md

```diff
@@ -70,6 +70,22 @@ Voiceless for all aircraft:
 
 All other aircraft coming from ARL CTA will be **Heads-up** Coordinated to WLM TCU.
 
+### Stereo Approaches (Fast Jet Aircraft)
```

Comment:
I reckon this is probably more appropriate for a **circuits** or **miscellaneous** section, as opposed to the coordination section

Reply:
Will move it there